tests/unit/test_mcp_gateway_evaluator_models.py

```
@@ -0,0 +1,127 @@
+        out = _summarize_tool_input({"command": long})
+        assert "...[truncated]" in out
+        # Truncated content + marker should be shorter than original
+        assert len(out) < len(long) + len("command=")
+
+    def test_handles_int_value(self) -> None:
+        out = _summarize_tool_input({"count": 42})
+        assert out == "count=42"
+
+    def test_redacts_sensitive_keys_in_nested_input(self) -> None:
+        # Verify Issue 1: nested sensitive keys are redacted
+        secret = "Bearer TEST_TOKEN"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "secret"

(S105)